src/client/scenes/universe/views/starsystemview.cpp

```diff
@@ -1332,7 +1332,22 @@ void SysStarSystemRenderer::DrawOrbit(const entt::entity& entity) {
     //    glm::quat{{0.f, 0, (float)body.axial}});
     // Draw orbit
     orbit_shader->SetMVP(transform, camera_matrix, m_app.Get3DProj());
-    orbit_shader->Set("color", glm::vec4(1, 1, 1, 1));
+
+    //Set the color of each orbit based on its distance from its center body
+    double dis = m_universe.get<common::components::types::Orbit>(entity).semi_major_axis;
+    double max_dis = m_universe.get<common::components::bodies::Body>(ref).SOI;
+    double inc = m_universe.get<common::components::types::Orbit>(entity).inclination;
+
+    float min_launch_dis = 8000;
+    dis = dis - min_launch_dis;
+    float r = log(dis) / log(max_dis);
+    float g = 1 - r;
+    float b = inc / 3.15;
```

Comment:
Inclination is [0, pi/2] so it should 1.57 instead of 3.15

Reply:
Okay